GCMtools/plotting.py

```diff
@@ -49,13 +57,115 @@ def isobaric_slice(self, ds, var_key, p):
         p : float
             Pressure level for the isobaric slice to be plotted, expressed in
             the units specified in the init-method.
+        time : int, optional
+            Timestamp that should be plotted. By default, the last time is
+            selected.
+        ax : matplotlib.axes.Axes, optional
+            The axis on which you want your plot to appear.
+        plot_windvectors : boolean, optional
+            If True (default), plot the horizontal wind vectors over the colour
+            mesh.
+        wind_kwargs : dict, optional
+            Additional keywords for the method plot_horizontal_wind.
+        cbar_kwargs : dict, optional
+            Additional keywords for the colorbar.
+        fs_labels : int, optional
+            Font size of the axis labels, 14 by default.
+        fs_ticks : int, optional
+            Font size of the tick labels, 11 by default.
```

Comment:
I (personally) do have a littlebit of a problem with that, because I usually set the label sizes prior using matplotlb rc to match the fontsize of the papertext.

Maybe we can have None as default, so that the default value from matplotlib is used?

Reply:
Alright, thanks for letting me know. This is different from what I do, but your way is more flexible, so I will put them to None by default.